server.js

```diff
@@ -0,0 +1,5 @@
+const http = require('http');
+const app = require('./lib/app');
+
+http.createServer(app)
+  .listen(7890);
```

Comment:
```suggestion
  .listen(PORT, () => console.log(`Server running on port ${PORT}`));
```
Having this added information is helpful if you want to visually confirm that your server is running (and on what port).